soroban-sdk-macros/src/derive_enum.rs

```diff
@@ -163,14 +163,6 @@ pub fn derive_type_enum(
             }
         }
 
-        impl #path::TryIntoVal<#path::Env, #enum_ident> for #path::RawVal {
```

Comment:
I think we still need the try into val's for cases where we assume every type has the try into val? Or does this get generated by a blanket?

Reply:
TryIntoVal always comes from the blanket TryIntoVal for TryFromVal.

Reply:
Sounds good to me.